fix(override-extend): update override-extend-variable module content (#211)
patternfly/training-scenarios: html-css/override-extend-variables/step2-instructions.md

Reviewer: sb Now that you have a card on your page, you can customize the styles to it.

Reviewer: sb:  all of the css values you use are first assigned to variables. It is recommended doing the same when making customizations in your application.

> ## Override PatternFly variables to achieve a new theme

Reviewer 2: Remove this line as we're not using these anymore

> Now that you have a card on your page, let's customize the styles applied to it.

Reviewer 2: Avoid using "you"

> 1) <strong>Add a new color variable to use in the card.</strong> Let's make the links in the card purple! In PatternFly, all of the css values you use are first assigned to variables. It is recommended doing the same when making customizations in your application.

Reviewer 2: Add a return statement after the first sentence. 
Remove "you"
`It is recommended doing the same when making customizations in your application.` --> change this sentence it doesn't sound like English

 
In `myapp.scss` inside of the `:root` block make a new variable name called `--my-app-card-theme--Color` and assign it to be purple: `#6200EE`.
 
<strong>Hint:</strong> 
```
  --my-app-card-theme--Color: #6200EE;
```
 

> 2) <strong>Override the global link color. </strong> In our current example of the card, the buttons in the card footer use the global link color. You can reassign the global link color variable to use our new custom color variable, so that all properties that use the global link color variable in the card will now use this color.

Reviewer 2: Remove instance of "you"
Add a return statement after the first sentence

 
Inside of the `.pf-c-card` block of `myapp.scss` set the global variable `--pf-global--link--Color` to use the value defined in the custom variable `--my-app-card-theme--Color`. 
 
<strong>Hint:</strong>
```
  --pf-global--link--Color: var(--my-app-card-theme--Color);

> ```

Reviewer: sb: So even though you are assigning a new value to the global variable, that value will only apply to the Card component<no comma> and not other components.

> ```

Reviewer: we'll sb you will

> <strong>Note:</strong> Variable definitions declared for a component `.pf-c-[component name]` class selector are scoped to that component. So even though you are assigning a new value to the global variable, that value will only apply to the Card component, and not other components.

Reviewer 2: Remove instance of "you"

> 3) <strong>Change the font-size of the title in the card header.</strong> Not all changes require custom css. Many components include modifier classes that enable you to customize the appearance of a component. For this update, you will use a modifier class available for the Title component. 

Reviewer 2: Add a return statement after the first sentence.

> 3) <strong>Change the font-size of the title in the card header.</strong> Not all changes require custom css. Many components include modifier classes that enable you to customize the appearance of a component. For this update, you will use a modifier class available for the Title component. 

Reviewer: remove you will start sentence with Use a modifier...

 
4) Search in `index.html` for `<div class="pf-c-card__header pf-c-title pf-m-md">` and replace `pf-m-md` with `pf-m-xl`.
 
<strong>Hint:</strong> `<div class="pf-c-card__header pf-c-title pf-m-xl">`
 

> <strong>When the server reloads,</strong> you should see something like this:

Reviewer 2: Why is this bolded? it hasn't been in any other step

> There is now a card on the page, customize the styles applied to it.

Reviewer 3: If I understand correctly, we're making a comment and then giving an action here. It should read, "There is now a card on the page. Customize the styles applied to it."

> Now that you have a card on your page, let's customize the styles applied to it.

Reviewer: ```suggestion
There is now a card on the page, customize the styles applied to it.
```

 
1) <strong>Add a new color variable to use in the card.</strong> 

>  Now make the links in the card purple! In PatternFly, all of the CSS values used are first assigned to variables. It is recommended doing the same when making customizations in the application.

Reviewer 3: Suggested change: Now make the links in the card purple.

> Let's make the links in the card purple! In PatternFly, all of the css values you use are first assigned to variables. It is recommended doing the same when making customizations in your application.

Reviewer 2: Let's still means let us - change this

> Let's make the links in the card purple! In PatternFly, all of the css values you use are first assigned to variables. It is recommended doing the same when making customizations in your application.

Reviewer: ```suggestion
 Now make the links in the card purple! In PatternFly, all of the CSS values used are first assigned to variables. It is recommended doing the same when making customizations in the application.
```

> In `myapp.scss` inside of the `:root` block make a new variable name called `--my-app-card-theme--Color` and assign it to be purple: `#6200EE`.

Reviewer 3: Suggested change: In `myapp.scss` inside of the `:root` block, make a new variable name called `--my-app-card-theme--Color`, and assign it to be purple: `#6200EE`. (punctuation fix)

 
<strong>Hint:</strong> 
```

>   --my-app-card-theme--Color: #6200EE;

Reviewer 2: extra space here before the line - which has an impact on formatting

```
 
2) <strong>Override the global link color. </strong> 

> In the current example of the card, the buttons in the card footer use the global link color. Reassign the global link color variable to use the new custom color variable so that all properties that use the global link color variable in the card will now use this color.

Reviewer 3: ```suggestion
In the current example of the card, the buttons in the card footer use the global link color. Reassign the global link color variable to use the new custom color variable. This way, all properties that use the global link color variable in the card will now use this color.
```

> In our current example of the card, the buttons in the card footer use the global link color. You can reassign the global link color variable to use our new custom color variable, so that all properties that use the global link color variable in the card will now use this color.

Reviewer: ```suggestion
In the current example of the card, the buttons in the card footer use the global link color. Reassign the global link color variable to use the new custom color variable so that all properties that use the global link color variable in the card will now use this color.
```

> Inside of the `.pf-c-card` block of `myapp.scss` set the global variable `--pf-global--link--Color` to use the value defined in the custom variable `--my-app-card-theme--Color`. 

Reviewer 3: ```suggestion
Inside of the `.pf-c-card` block of `myapp.scss`, set the global variable `--pf-global--link--Color` to use the value defined in the custom variable `--my-app-card-theme--Color`. 
```

 
<strong>Hint:</strong>
```
  --pf-global--link--Color: var(--my-app-card-theme--Color);
```
 

> <strong>Note:</strong> Variable definitions declared for a component `.pf-c-[component name]` class selector are scoped to that component. So even though you are assigning a new value to the global variable, that value will only apply to the Card component, and not other components.

Reviewer: ```suggestion
<strong>Note:</strong> Variable definitions declared for a component `.pf-c-[component name]` class selector are scoped to that component. So even though a new value is being assigned to the global variable, that value will only apply to the card component and not other components.
```

> 3) <strong>Change the font-size of the title in the card header.</strong> 

Reviewer 3: Correct!

> Not all changes require custom css. Many components include modifier classes that enable you to customize the appearance of a component. For this update, you will use a modifier class available for the Title component. 

Reviewer: ```suggestion
Not all changes require custom CSS. Many components include modifier classes that enable the customization of the component appearnance. For this update, you use a modifier class available for the Title component. 
```

> 4) Search in `index.html` for `<div class="pf-c-card__header pf-c-title pf-m-md">` and replace `pf-m-md` with `pf-m-xl`.

Reviewer 2: Needs to be bolded

 
<strong>Hint:</strong> `<div class="pf-c-card__header pf-c-title pf-m-xl">`

>  

Reviewer 2: instead of server use the word preview

> When the server reloads, you should see something like this:

Reviewer: ```suggestion
When the preview reloads, it should look something like this:
```
